vllm/platforms/__init__.py

```diff
@@ -217,8 +217,11 @@ def __getattr__(name: str):
             global _init_trace
             _init_trace = "".join(traceback.format_stack())
         return _current_platform
-    else:
+    elif name in globals().keys():
```

Comment:
```suggestion
    elif name in globals():
```
It is unnecessary to call `.keys()`

Reply:
Thanks for pointing that!